docs/cosmos-stack-overview.mdx

```diff
@@ -0,0 +1,503 @@
+---
+title: "Cosmos Stack Developer Documentation"
+description: "The most widely-adopted and battle-tested Layer 1 blockchain technology stack, in production with 200+ chains today"
+mode: "custom"
+---
+
+import { TopicCard, TopicCardGrid } from '/snippets/topic-card-component.jsx';
+
+<div style={{
+  minHeight: '100vh',
+  background: '#000000',
+  color: '#F1F1F1',
+  fontFamily: 'system-ui, -apple-system, sans-serif',
+}}>
+  {/* Banner */}
+  <div style={{
+    minHeight: '25vh',
+    background: 'linear-gradient(180deg, #0E0E0E 0%, #000000 100%)',
+    padding: '60px 40px',
+    borderBottom: '1px solid #cccccc20',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+  }}>
+    <h1 style={{
+      fontSize: '48px',
+      fontWeight: '700',
+      marginBottom: '16px',
+      color: '#F1F1F1',
+      textAlign: 'center',
+    }}>
+      Cosmos Stack Developer Documentation
+    </h1>
+    <p style={{
+      fontSize: '18px',
+      maxWidth: '800px',
+      textAlign: 'center',
+      color: '#cccccc',
+      lineHeight: '1.6',
+    }}>
+      The most widely-adopted and battle-tested Layer 1 blockchain technology stack, in production with 200+ chains today
+    </p>
+  </div>
+
+  {/* Content */}
+  <div style={{
+    maxWidth: '1200px',
+    margin: '0 auto',
+    padding: '60px 40px',
+  }}>
+    {/* Introduction */}
+    <section style={{ marginBottom: '80px', paddingBottom: '80px', borderBottom: '1px solid #cccccc10' }}>
+      <h2 style={{
+        fontSize: '32px',
+        fontWeight: '700',
+        marginBottom: '20px',
+        color: '#F1F1F1',
+      }}>
+        Introduction
+      </h2>
+      <p style={{
+        fontSize: '18px',
+        lineHeight: '1.8',
+        color: '#cccccc',
+        marginBottom: '16px',
+      }}>
+        The Cosmos stack is the most widely-adopted and battle-tested Layer 1 blockchain technology stack, in production with <strong>200+ chains</strong> today. It is performant, customizable, and EVM-compatible, offering engineers full end-to-end control of their blockchain infrastructure and implementation.
+      </p>
+      <p style={{
+        fontSize: '18px',
+        lineHeight: '1.8',
+        color: '#cccccc',
+      }}>
+        The stack is modular by design—use pre-built code modules or integrate custom features and logic for your use case, compliance needs, governance, consensus, and performance requirements. This documentation explains the open-source stack components: <strong>Cosmos SDK</strong>, <strong>CometBFT</strong>, <strong>IBC</strong>, and <strong>Cosmos EVM</strong>.
```

Comment:
I would also order it same as: https://github.com/cosmos/docs/pull/104/files#r2446621701